src/framework/audio/engine/internal/mixerchannel.cpp

```
@@ -26,30 +26,35 @@
 }
 
 MixerChannel::MixerChannel(const TrackId trackId, const OutputSpec& outputSpec,
-                           const IGetPlaybackPosition* getPlaybackPosition)
-    : MixerChannel(trackId, outputSpec, nullptr, getPlaybackPosition)
+                           PlayheadPositionPtr playheadPosition)
+    : MixerChannel(trackId, outputSpec, nullptr, playheadPosition)
 {
     ONLY_AUDIO_ENGINE_THREAD;
 }
 
+void MixerChannel::setPlayheadPosition(PlayheadPositionPtr playheadPosition)
```

Cppcheck (2.20.0)

[style] The function 'multiplySamples' is never used.

(unusedFunction)